control_protocol.md

```diff
@@ -58,10 +62,10 @@ A message consists of 4 or more frames.
 
 #### Directory
 
-Each Coordinator shall have a list of the Components (including other Connectors) connected to it.
-This is its _Directory_.
+Each Coordinator shall have a list of the Components (including other Coordinators) connected to it.
+This is its _local Directory_.
 
-The _Glossary_ is the combination of the Directories of all Coordinators in a Network.
+The _global Directory_ is the combination of the Directories of all Coordinators in a Network.
```

Comment:
```suggestion
The _global Directory_ is the union of the Directories of all Coordinators in a Network.
```
More precise term here.

Reply:
However, with this definition for the global one, this basically becomes an unknowable quantity, as its "true" content is distributed over the Network, so in principle one single Coordinator could not even send it elsewhere.

Reply:
```suggestion
The _global Directory_ is a Coordinator's copy of the union of the local Directories of all Coordinators in a Network.
```

Is this better, as it adds the notion, that it is a local copy, which might not coincide perfectly with the mathematical union (unknowable).